src/Transformer/UuidHexTransformer.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace Diaclone\Transformer;
+
+use Diaclone\Resource\ResourceInterface;
+use Ramsey\Uuid\Uuid;
+
+class UuidHexTransformer extends UuidTransformer
+{
+    public function transform(ResourceInterface $resource)
+    {
+        /** @var Uuid $value */
+        $uuid = $this->getPropertyValueFromResource($resource);
+
+        return $uuid->getHex();
```

Comment:
this assumes the value is UUID. what if it's a UUID string?

Reply:
I agree but also we choose what we strap transformers on, if its a string then we can use string transformer?

Reply:
it's the same scenerio that was encountered with enums. technically, it should never be a string. but just in case :)

Reply:
With it being a class, it could come back as null, so yeah, I need to handle it the same way as the enum, I'll add that. If its a string, then use `UuidTransformer` since it can handle a string or Uuid object.